website/highlights/2020-07-09-add-musl-and-glibc-support-to-install-sh.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
In general, I'd like to rework this highlight to focus on performance benefit, which is a huge value add that I'm afraid gets lost in the current form. Basically:

- [ ] Update the title to something performance-focused.
- [ ] Swap `type: breaking change` for `type: performance`.
- [ ] Update the body to focus on the performance benefit, feel free to mention how we did it.
- [ ] Remove the `Upgrade Guide` section since this is not a breaking change.

What do you think?

Reply:
So this is a breaking change. We're changing the way we link to link to the system. 

It breaks compatibility  with old systems, it may require manual intervention in edge cases.

This means this is no longer valid on glibc packages: 
![image](https://user-images.githubusercontent.com/130903/87727379-b7faf600-c775-11ea-94ae-e030ad3784b4.png)

Reply:
Maybe I can include some performance difference measurements?